bot/commands/commands.js

```diff
@@ -0,0 +1,23 @@
+﻿const messages = require('../../messages/format.js');
+
+const commands = (msg) =>
```

Comment:
With only one parameter parens aren't required here.

Reply:
This is true but it also does not affect the execution of the execution of the code itself but makes it easier to declare no parameter or multiple parameters if needed.

() => { ... } // no parameter
(x) => { ... } // one parameter, an identifier
(x, y) => { ... } // several parameters